src/main/config/proxy-compose.yaml.ctmpl

```diff
@@ -7,15 +7,11 @@ proxy:
   {{.Data.proxy_ports}}
   {{.Data.proxy_volumes}}
   environment:
-    AUTH_TYPE: AuthType None
-    AUTH_TYPE2: AuthType oauth20
     CALLBACK_URI: {{.Data.env_callback_uri}}
-    LOG_LEVEL: warn
-    OIDC_CLAIM: Require all granted
```

Comment:
We are currently getting the user's email address from the request header via the "OIDC_CLAIM_email" value. That email becomes the primary thing we rely on to keep track of our users. Is that going away?